src/constants/welcomeModalProps.ts

```diff
@@ -0,0 +1,73 @@
+const newMainlineCommitsUser = [
+  {
+    img: "mainline_commits/carousel_1_05_06.png",
+    title: "Welcome to the new mainline commits experience!",
+    description:
+      "We designed this UI to make navigation faster and more intuitive. By default it displays failed tasks, but you can customize this view with filters.",
+    href:
+      "https://app.tango.us/app/workflow/Evergreen--Onboarding-guide-for-the-new-project-health-page--7b74b28c80f448869a01730a450bc246",
+  },
+  {
+    video: "mainline_commits/carousel_2_05_06.webm",
+    title: "Get a quick overview of your project's health",
+    description:
+      "A graph at the top of the page gives you an overview of the project&apos;s health. You can toggle the graph between an absolute number  and percentage.",
```

Comment:
None of the apostrophes in this file are showing properly for me... 🤔 I think if you just use a normal `'`, it should work

<img width="503" alt="Screen Shot 2022-05-09 at 11 49 41 AM" src="https://user-images.githubusercontent.com/47064971/167449795-a3800827-6305-4a89-bd2d-f9c160aa87b3.png">